packages/trace-viewer/src/ui/networkTab.tsx

```diff
@@ -14,25 +14,198 @@
  * limitations under the License.
  */
 
+import type { Entry } from '@trace/har';
+import { ListView } from '@web/components/listView';
 import * as React from 'react';
+import type { Boundaries } from '../geometry';
 import type * as modelUtil from './modelUtil';
-import { NetworkResource } from './networkResourceDetails';
 import './networkTab.css';
-import type { Boundaries } from '../geometry';
+import { NetworkResourceDetails } from './networkResourceDetails';
+import { bytesToString, msToString } from '@web/uiUtils';
+
+const NetworkListView = ListView<Entry>;
+
+type Filter = 'status' | 'method' | 'file' | 'time' | 'size' | 'content-type';
```

Comment:
Why is this called `Filter` when it is a sorting field?

Reply:
I'll rename in a follow-up.